kolibri/core/assets/src/views/UpdateNotification.vue

```diff
@@ -0,0 +1,43 @@
+<template>
+
+  <KModal
+    v-if="displayed"
+    size="large"
+    :submitText="$tr('okButtonLabel')"
+    :title="$tr('updateModalHeader')"
+  >
+    <p>{{ updateMessageByLang }}</p>
+    <KExternalLink :href="link" :text="linkTextByLang" />
+    <KCheckbox :label="$tr('hideNotificationLabel')" />
+
+  </KModal>
+
+</template>
+
+
+<script>
+
+  import KExternalLink from 'kolibri.coreVue.components.KExternalLink';
+  import KModal from 'kolibri.coreVue.components.KModal';
+  import KCheckbox from 'kolibri.coreVue.components.KCheckbox';
+
+  export default {
+    name: 'UpdateNotification',
+    components: {
+      KCheckbox,
+      KExternalLink,
+      KModal,
+    },
+    data() {
+      return {
+        displayed: false,
+      };
+    },
+    $trs: {
+      updateModalHeader: 'Upgrade available',
```

Comment:
I think this is not technically necessary because the [API sends the title](https://docs.google.com/document/d/1h5hyHttxTrZpQq7j0sPCBLNlcV7PcaJ5CrtlJvVZBdM/edit#heading=h.oa4qzi6krct0) (which can be non-upgrade-related)